gadgets/src/batched_is_zero.rs

```diff
@@ -89,6 +89,14 @@ impl<F: FieldExt, const N: usize> BatchedIsZeroChip<F, N> {
                 (F::one(), F::zero())
             }
         });
+
+        // annotate columns
```

Comment:
Unfortunately these annotations are in an `assign` function that is called many times.  I would suggest to follow this approach https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1167/files so that the annotation is only added once in the region.  The approach also allows you to prefix the names with the circuit name.

Reply:
fixed in [1b5aa41](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/1b5aa4193380414a8378031399be7a6eddfac11a)